textdb/textdb-perftest/src/main/java/edu/uci/ics/textdb/perftest/nlpextractor/NlpExtractorPerformanceTest.java

```diff
@@ -122,8 +122,8 @@ public static void matchNLP(IDataStore dataStore, NlpTokenType tokenType, Analyz
 		QueryParser queryParser = new QueryParser(MedlineIndexWriter.ABSTRACT_ATTR.getFieldName(), analyzer);
 		Query query = queryParser.parse(DataConstants.SCAN_QUERY);
 
-		DataReaderPredicate dataReaderPredicate = new DataReaderPredicate(dataStore, query, DataConstants.SCAN_QUERY,
-				analyzer, attributeList);
+		DataReaderPredicate dataReaderPredicate = new DataReaderPredicate(query, DataConstants.SCAN_QUERY,
```

Comment:
Try to see if we can combine "query" and "DataConstants.SCAN_QUERY" to a single query parameter.